casanovo/config.py

```
@@ -27,185 +27,139 @@
 )
 
 
+def _to_python(val: Any) -> Any:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `val`

(ANN401)